src/router.ts

```diff
@@ -15,6 +15,17 @@ const routes = [
         component: Home,
       },
     ],
+  },
+  {
+    path: '/',
+    component: DefaultVue,
```

Comment:
@cesarvh - not a merge blocker but perhaps we want `DefaultView` instead of `DefaultVue`?